bin/stacks/routing-api-stack.ts

```diff
@@ -299,7 +299,8 @@ export class RoutingAPIStack extends cdk.Stack {
         },
       }),
       threshold: 40,
-      evaluationPeriods: 2,
+      evaluationPeriods: 3,
+      datapointsToAlarm: 2
```

Comment:
this makes the alarm even more sensitive than before, no?
previously it needs two consecutive periods evaluate to over threshold, now it only needs any two out of the latest three.

Reply:
yeah you're right

Reply:
changed to evaulationPeriods = 5 and datapointsToAlarm = 4